x-pack/solutions/security/plugins/security_solution/public/flyout/attack_details/types.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FlyoutPanelProps } from '@kbn/expandable-flyout';
+import type { AttackDetailsPanelKey } from './constants/panel_keys';
+
+export interface AttackDetailsProps extends FlyoutPanelProps {
+  key: typeof AttackDetailsPanelKey;
+  params?: {
+    id: string;
```

Comment:
What do you think about using `attackId` here? It will be a bit more clear to the developers who will use the flyout api